server/json/user_json.jsx

```diff
@@ -18,6 +18,7 @@ export default function useUserJson(app) {
 
         if(chainAccount) {
             user = chainAccount;
+            user.json_metadata = user.json_metadata ? JSON.parse(user.json_metadata) : "";
```

Comment:
@mcifani will this barf if the user has put invalid json into their profile? should this be in a try/catch?